src/components/vehicleReference/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, ReactNode } from 'react';
+
+import {
+  AspectRatio,
+  Box,
+  chakra,
+  useMultiStyleConfig,
+} from '@chakra-ui/react';
+
+import Stack from '../stack';
+
+interface Props {
+  image: ReactNode;
+  vehicleTitle: string;
+  price: string;
+  sellerName: string;
+  sellerAddress: string;
+}
+
+const VehicleReference: FC<Props> = ({
+  image,
+  vehicleTitle,
+  price,
+  sellerName,
+  sellerAddress,
+}) => {
+  const styles = useMultiStyleConfig(`VehicleReference`);
+
+  return (
+    <Stack direction={{ xs: 'row', lg: 'column' }} spacing="md">
+      <AspectRatio minW="2xl" ratio={4 / 3} borderRadius="sm" overflow="hidden">
+        {image}
+      </AspectRatio>
+      <Stack spacing={{ xs: 'xs', lg: 'md' }} justify="center">
+        <chakra.h1 __css={styles.carTitle}>{vehicleTitle}</chakra.h1>
```

Comment:
Since you are using an h1, should we wrap the entire component in something like an aside or article to avoid having duplicated h1 elements on the page?